recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# Make sure -fPIC is not in CXXFLAGS (that some conda packages may
+# add):
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Make sure that llvm-config and clang++ are from host environment,
+# otherwise UdfTest will fail:
+export PATH=$PREFIX/bin:$PATH
+
+export CC=${PREFIX}/bin/clang
+export CXX=${PREFIX}/bin/clang++
+
+# Resolves cmake message `It appears that you have Arrow < 0.10.0`:
+export CFLAGS="$CFLAGS -pthread"
+export LDFLAGS="$LDFLAGS -pthread -lrt -lresolv"
+
+# fixes `undefined reference to
+# `boost::system::detail::system_category_instance'`:
+export CXXFLAGS="$CXXFLAGS -DBOOST_ERROR_CODE_HEADER_ONLY"
+
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_LIBRARY_PATH=$CUDA_HOME/lib64/stubs"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME"
+
+mkdir -p build
+cd build
+
+cmake -Wno-dev \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_BUILD_TYPE=release \
+    -DMAPD_DOCS_DOWNLOAD=off \
+    -DENABLE_AWS_S3=off \
+    -DENABLE_CUDA=on \
+    -DENABLE_FOLLY=off \
+    -DENABLE_JAVA_REMOTE_DEBUG=off \
+    -DENABLE_PROFILER=off \
+    -DENABLE_TESTS=on  \
+    -DPREFER_STATIC_LIBS=off \
+    $EXTRA_CMAKE_OPTIONS \
+    ..
+
+(which lscpu && which grep && which awk && which bc)
+if [ $? -eq 0 ]; then
+    export CORES_PER_SOCKET=`lscpu | grep 'Core(s) per socket' | awk '{print $NF}'`
+    export NUMBER_OF_SOCKETS=`lscpu | grep 'Socket(s)' | awk '{print $NF}'`
+    export NCORES=`echo "$CORES_PER_SOCKET * $NUMBER_OF_SOCKETS" | bc`
+    make -j $NCORES
```

Comment:
You can remove the `if` condition and keep the `else` condition. `CPU_COUNT is set by conda-build using a similar strategy and the user can override it by setting the env var before running conda-build.

Reply:
Yes, I agree. The `if` condition was useful only for testing conda-build locally. For instance, on a 24/48 cores/threads processor system, the CPU_COUNT is 48 while using 24 is sufficient (hyperthreading is useless for build tasks) and required (build will fail due to high RAM usage).

Reply:
For testing locally, you can set `CPU_COUNT` env variable before running conda-build.

Reply:
Yes, I have removed the `if` block.